client/src/components/selects/addExtensionSelect.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
```

Comment:
How different is this file from `addResourceSelect`? Do you think it would be easy to merge these as a generic component? If it's too long, we can put it in a TODO list

Reply:
it's quite different from addResourceSelect because it not renders profiles but it could be made generic (probably in the same PR as the generic upload)